fairlearn/datasets/_fetch_compas.py

```diff
@@ -0,0 +1,139 @@
+    juv_fel_count          number of juvenile felony convictions
+    juv_misd_count         number of juvenile misdemeanor convictions
+    juv_other_count        number of other juvenile convictions
+    priors_count           number of prior convictions
+    age_cat_25-45          1 if defendant's age >=25 and <45, otherwise 0
+    age_cat_Greaterthan45  1 if defendant's age >=45, otherwise 0
+    age_cat_Lessthan25     1 if defendant's age <25, otherwise 0
+    race_African-American  1 if defendant's race African-American, otherwise 0
+    race_Caucasian         1 if defendant's race Caucasian, otherwise 0
+    c_charge_degree_F      1 if charged for felony, otherwise 0
+    c_charge_degree_M      1 if charged for misdemeanor, otherwise 0
+    =====================  ======================================================================
+
+    Parameters
+    ----------
+    cache : boolean, default=True
+        Whether to cache downloaded datasets using joblib.
+
+    data_home : optional, default: None
+        Specify another download and cache folder for the datasets. By default
+        all fairlearn data is stored in '~/.fairlearn-data' subfolders.
+
+    as_frame : boolean, default=False
+        If True, the data is a pandas DataFrame including columns with
+        appropriate dtypes (numeric, string or categorical). The target is
+        a pandas DataFrame or Series depending on the number of target_columns.
+        The Bunch will contain a ``frame`` attribute with the target and the
+        data. If ``return_X_y`` is True, then ``(data, target)`` will be pandas
+        DataFrames or Series as describe above.
+
+    return_X_y : boolean, default=False.
+        If True, returns ``(data.data, data.target)`` instead of a Bunch
+        object.
+
+    warn : boolean, default=True.
+        If True, it raises an extra warning to make users aware of the unfairness
+        aspect of this dataset.
+
+
+    Returns
+    -------
+    dataset : :class:`~sklearn.utils.Bunch`
+        Dictionary-like object, with the following attributes.
+
+        data : ndarray, shape (5278, 13)
+            Each row corresponding to the 13 feature values in order.
+            If ``as_frame`` is True, ``data`` is a pandas object.
+        target : numpy array of shape (5278,)
+            Each value indicates whether a defendant was charged again within two years after the
+            current charge.
+            If ``as_frame`` is True, ``target`` is a pandas object.
+        feature_names : list of length 13
+            Array of ordered feature names used in the dataset.
+        DESCR : string
+            Description of the COMPAS dataset.
+
+    (data, target) : tuple of (numpy.ndarray, numpy.ndarray) or (pandas.DataFrame, pandas.Series)
+        if ``return_X_y`` is True and ``as_frame`` is False
+
+    (data, target) : tuple of (pandas.DataFrame, pandas.Series)
+        if ``return_X_y`` is True and ``as_frame`` is True
+
+    See Also
```

Comment:
What's this for? I'm not sure I understand the purpose.

Reply:
I guess this is just a habit of mine when writing docstrings, but you can think of it as a recommender of "other functions you might like". 

Talking docs, I think they might be easier to read if we move towards overview table + separate pages for each class/function (i.e. like in scikit-learn's docs). What do you think?

Reply:
Definitely prefer overview table! This approach won't scale long-term :-)
It's fine if we do that in a separate PR, though, I'm not too particular about it.